AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_OcclusionCullingPlaneAdded.py

```diff
@@ -153,4 +156,4 @@ def AtomEditorComponents_Occlusion_Culling_Plane_AddedToEntity():
 
 if __name__ == "__main__":
     from editor_python_test_tools.utils import Report
-    Report.start_test(AtomEditorComponents_Occlusion_Culling_Plane_AddedToEntity)
+    Report.start_test(AtomEditorComponents_occlusion_culling_plane_AddedToEntity)
```

Comment:
```suggestion
    Report.start_test(AtomEditorComponents_OcclusionCullingPlane_AddedToEntity)
```

To match Scott's above suggestion, you'll need to update the formatting to match your function definition.

Reply:
Fixed!